### ISSUE TITLE
Move to Ionide org

### ISSUE THREAD

**Issue by alfonsogarciacaro** [fable-compiler/repl-legacy#18]
Right after FableConf, I agreed with Krzystof that we'd name the new project Ionide-Fable and he [created a repo for us](https://github.com/ionide/ionide-fable) in Ionide org, but I forgot to move the code there 😅 

Moving the REPL to Ionide org may help create synergies from both projects: sharing more code, getting help from Ionide contributors, etc. What do you think @MangelMaxime @ncave?

**Comment by MangelMaxime** [comment]
Sure I am OK with that 😊

What will we share the whole project? Only part of it?

**Comment by alfonsogarciacaro** [comment]
I guess the easiest way is to transfer the whole repo, but I'm not sure if that will break the web address we just announced. I'll look into that :+1:

**Comment by MangelMaxime** [comment]
We also have hard-coded url in the generator.fs file

**Comment by ncave** [comment]
@alfonsogarciacaro I don't really have an opinion either way on Fable org vs the Ionide org, whichever org you think will add more contributors or visibility in the long run.

**Comment by voronoipotato** [comment]
I think putting a fable repl on ionide seems fine but I hope this doesn't mean removing the fable.io site or the fable.io/repl . That I think would be an unnecessary step backwards.

**Comment by alfonsogarciacaro** [comment]
Yeah, I was thinking something like that. I'd like to automate REPL updates with Fable/FCS updates, and going through too many repos can make this difficult.
